CONTRIBUTING.rst

```diff
@@ -0,0 +1,104 @@
+.. _contributing:
+
+Contributing
+------------
+
+If you find errors, omissions, inconsistencies or other things that need
+improvement, please create an issue or a pull request at
+https://github.com/sfstoolbox/theory/.
+Contributions are always welcome!
+
+
+Installation
+^^^^^^^^^^^^
+
+In order to build the theory section locally you should get the
+newest development version from Github_ and install the needed dependencies
+with::
+
+   git clone https://github.com/sfstoolbox/theory.git
+   cd theory
+   python3 -m pip install --user -e .
+
+.. _Github: https://github.com/sfstoolbox/theory/
+
+This way, your installation always stays up-to-date, even if you pull new
+changes from the Github repository.
+
+
+Building the Documentation
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you make changes to the documentation, you can re-create the HTML and PDF
+pages using Sphinx_.
+
+To create the HTML pages, use::
+
+   make html-preview
+
+The generated files will be available in the directory
+``build/sphinx/html-preview/``.
+To create the PDF pages, use::
+
+    make latexpdf
+
+The generated files will be available in the directory ``build/sphinx/latex/``.
```

Comment:
That should actually be `_build/latex/`.

Reply:
Corrected.